lib/administrate/page/collection.rb

```diff
@@ -3,6 +3,11 @@
 module Administrate
   module Page
     class Collection < Page::Base
+      def initialize(dashboard, resources, options = {})
+        super(dashboard, options)
+        @resources = resources
+      end
```

Comment:
Again, agree with changing this initializer :+1: We may want to go one step further and change it at `Page::Base`, but let's leave that for now.